modules/rating/src/main/Glicko.scala

```diff
@@ -35,6 +35,7 @@ case class Glicko(
 
```

Comment:
Looks like you're squashing deviation after a sanity check that could fail?

Reply:
Yeah you're right. I should probably remove the `deviation < 1000` check which no longer makes sense.

Reply:
if that's your argument, then then you don't need to be squashing deviation to maxDeviation...